index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+  <title>My Portofolio</title>
+</head>
+<body>
+
+  <div class="main-content">
+  <h1>Hey There.</h1>
+  <h1>I'm Wycliff</h1>
+  <h2>I'm software developer!</h2>
+  <h3>I can help you build a project, feature or website. Look</h3>
+   <h3> some of the works and experience!If you like what </h3>
+    <h3>you see and have a project you need coded,</h3>
+    <h3>don't hesitate to contact me!</h3>
+    </div>
+
+<div class="fa">
+<a href="#" class="fa fa-facebook"></a>
+<a href="#" class="fa fa-twitter"></a>
+<a href="#" class="fa fa-linkedin"></a>
+<a href="#" class="fa fa-github"></a>
+</div>
```

Comment:
- [ ] The list of icons should be grouped with a `<ul>` tag as this is the best way to markdown a list of items 😎